src/shared-components/PaginationControl/PaginationControl.tsx

```
@@ -0,0 +1,251 @@
+      setJumpInput(clampedValue.toString());
+    } else {
+      // Revert to current page on invalid input
+      setJumpInput(safePage.toString());
+    }
+  };
+
+  return (
+    <div
+      ref={paginationRef}
+      className={styles.paginationWrap}
+      role="navigation"
```

Biome (2.4.6)

[error] The elements with this role can be changed to the following elements:
<nav>

(lint/a11y/useSemanticElements)